### PR TITLE
Defaulting EQLDIMS item 2 in testcases, since it was not previously used

### PR DESCRIPTION
**Pull request by vkip**

### CODE REVIEW

File: network/TMP.DATA
Comment (bska) [inline network/TMP.DATA:1]: This file (`network/TMP.DATA`) is new in this PR.  Is that intentional?

**Comment by bska** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
A quick and high-level reading of the PR suggests that this is good and I've started a build check.  There is just one thing that stands out here, the addition of the new file `network/TMP.DATA`.  Is that a leftover from some automated file handling?

File: network/TMP.DATA
Comment (vkip) [reply network/TMP.DATA:1]: Oops, no - too many files.. fixing now.

**Comment by bska** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master once the build check completes.

**Comment by bska** [comment]
> This looks good to me now and I'll merge into master once the build check completes.

The build check is green so I'll merge into master.  Thanks a lot!

**Pull request merged**